ci_build_pipelines/publish-restler.yml

```diff
@@ -0,0 +1,64 @@
+trigger:
+  batch : true
+  branches:
+    include:
+      - 'main'
+
```

Comment:
I just left a similar comment in the PBI - can you please help us understand when this is going to be triggered?

Reply:
The pipeline would only be triggered manually. We only want to create new public images when you decide to release a new version.

Reply:
Instead of being manually triggered, you could use a different trigger.

This will run the workflow when a release has been published.
```
on:
  release:
    types: [published]
```

https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#release

Reply:
For now restler isn't using actions. It's something to consider though.